translations/views.py

```diff
@@ -28,31 +31,55 @@ def get(self, request):
 
 
 class NewTranslationForm(forms.Form):
-    label = forms.CharField(max_length=128)
-    default_message = forms.CharField(widget=forms.Textarea(attrs={'name': 'text', 'rows': 3, 'cols': 50}))
+    label = forms.CharField(max_length=128, widget=forms.TextInput(
+        attrs={'class': 'input'}))
+    default_message = forms.CharField(widget=forms.Textarea(
+        attrs={'name': 'text', 'rows': 3, 'cols': 50, 'class': 'textarea'}))
 
 
 @login_required(login_url='/admin/login')
 @staff_member_required
 def admin_view(request):
     if request.method == 'GET':
         translations = Translation.objects.all()
+        # Display 50 translations per page
+        paginator = Paginator(translations, 50)
+        page_number = request.GET.get('page')
+        page_obj = paginator.get_page(page_number)
+
+        for translation in page_obj:
+            if re.search(r'^\w+\..+\_\d+\-\w+$', translation.label):
+                translation.model_name = translation.used_by
+                translation.field_name = translation.label.split('-')[-1]
+                translation.entry_id = int(
+                    re.search(r'_(\d+)-', translation.label).group(1))
+                translation.abbr_name = re.search(
+                    r'\.(.+?)_\d+', translation.label).group(1)
```

Comment:
I think that you can get all of this info by getting the reverse model on all of these. For example, I used `name_abbreviated` in the translations export.

Reply:
> I used name_abbreviated in the translations export

I am not sure where this is. 

I don't think the field name and object's entry ID that I am parsing from the label can be found in the reverse model. I am sure you know this better than me, but the label is generated with the field name and entry ID during the addition of a translation object, so it'll have to be parsed manually like this from the label.